src/sentry/static/sentry/app/components/modals/inviteMembersModal/index.tsx

```diff
@@ -0,0 +1,391 @@
+      return (
+        <StatusMessage>
+          <LoadingIndicator mini relative hideMessage size={16} />
+          {t('Sending organization invitations...')}
+        </StatusMessage>
+      );
+    }
+
+    if (complete) {
+      const statuses = Object.values(inviteStatus);
+      const sentCount = statuses.filter(i => i.sent).length;
+      const errorCount = statuses.filter(i => i.error).length;
+
+      const invites = <strong>{tn('%d invite', '%d invites', sentCount)}</strong>;
+
+      return (
+        <StatusMessage status="success">
+          <InlineSvg src="icon-checkmark-sm" size="16px" />
+          {errorCount > 0
+            ? tct('Sent [invites], [failed] failed to send.', {
+                invites,
+                failed: errorCount,
+              })
+            : tct('Sent [invites]', {invites})}
+        </StatusMessage>
+      );
+    }
+
+    if (this.hasDuplicateEmails) {
+      return (
+        <StatusMessage status="error">
+          <InlineSvg src="icon-warning-sm" size="16px" />
+          {t('Duplicate emails between invite rows.')}
+        </StatusMessage>
+      );
+    }
+
+    return null;
+  }
+
+  render() {
+    const {Footer, closeModal, organization} = this.props;
+    const {pendingInvites, sendingInvites, complete, inviteStatus, member} = this.state;
+
+    const disableInputs = sendingInvites || complete;
+
+    return (
+      <React.Fragment>
+        <Heading>
+          <InlineSvg src="icon-mail" size="36px" />
+          {t('Invite New Members')}
+        </Heading>
+        <Subtext>
+          {t('Invite new members by email invitation to join your Organization.')}
+        </Subtext>
+
+        <InviteeHeadings>
+          <div>{t('Email addresses')}</div>
+          <div>{t('Role')}</div>
+          <div>{t('Add to team')}</div>
+        </InviteeHeadings>
+
+        {pendingInvites.map(({emails, role, teams}, i) => (
+          <StyledInviteRow
+            key={i}
+            disabled={disableInputs}
+            emails={[...emails]}
+            role={role}
+            teams={[...teams]}
+            roleOptions={member && member.roles}
+            teamOptions={organization.teams}
+            inviteStatus={inviteStatus}
+            onRemove={() => this.removeInviteRow(i)}
+            onChangeEmails={opts => this.setEmails(opts.map(v => v.value), i)}
+            onChangeRole={({value}) => this.setRole(value, i)}
+            onChangeTeams={opts => this.setTeams(opts.map(v => v.value), i)}
+            disableRemove={disableInputs || pendingInvites.length === 1}
+          />
+        ))}
+
+        <AddButton priority="link" onClick={this.addInviteRow} icon="icon-circle-add">
+          {t('Add another')}
+        </AddButton>
+
+        <Footer>
+          <FooterContent>
+            <div>{this.statusMessage}</div>
+
+            {complete ? (
+              <React.Fragment>
+                <Button label="send more" size="small" onClick={this.reset}>
```

Comment:
Ah, I was using this for tests

Reply:
Removed label. Used `data-test-id` for tests.

Reply:
Oops, I should have clarified further - if `children` is a string, `<Button>` uses it for `aria-label`, otherwise you can specify `label` if e.g. you have only an icon for the button contents. I think in tests it's a good habit to use `aria-label` when possible as a query selector so that we're kind of testing for accessibility in the process

Reply:
Ah I see what you're saying, yeah I can convert these over to aria-label selectors. cool

Reply:
Actually I think my opinion on this has shifted a bit. It feels pretty weird to use longer button labels as selectors, and is especially harder to debug in jest. I think I'd like to opt to keep these as data-test-id for now. What do you think?

Reply:
How does it make debugging in jest harder?

The selectors can be longer, but it also reflects what the user sees (or hears) and doesn't require an additional test-only attribute.

Reply:
(I'm fine with the tests as is, just trying to further understand)

Reply:
> How does it make debugging in jest harder?

If the text is dynamic, it's a bit harder to figure out exactly what the selector should be. For example I wanted to test that the add button had the correct number added to it. Easy to see the diff when the selector for it is know, but if the number changes, now the selector fails and it just says `cannot use .text on 0 elements`.